meadow-client/src/Data/BigInteger.purs

```diff
@@ -0,0 +1,54 @@
+module Data.BigInteger (BigInteger, fromInt, fromString) where
```

Comment:
I think this warrants a module-level docstring explaining briefly why we need to wrap `BigInt`.